oidc_cli/token_getter.go

```diff
@@ -42,7 +38,7 @@ func GetToken(ctx context.Context, clientID string, issuerURL string, clientOpti
 		return nil, errors.Wrap(err, "Unable to create client")
 	}
 
-	storage, err := getStorage(clientID, issuerURL)
+	storage, err := storage.getStorage(clientID, issuerURL)
```

Comment:
when moving to another package, lowerCased entities are not exported to other packages. s/getStorage/GetStorage/ would make it exportable across package boundaries